apps/ledger-live-mobile/src/screens/FirmwareUpdate/RestoreStepDenied.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Text, Flex, Icons, IconBadge } from "@ledgerhq/native-ui";
+import { Device } from "@ledgerhq/types-devices";
+import { TFunction } from "i18next";
+import { getDeviceModel } from "@ledgerhq/devices";
+import Button from "../../components/Button";
```

Comment:
Use `Button` from `components/wrappedUi/Button` instead, this one should be deprecated because the props are not matching the design system.
(wrappedUi components are just wrappers that adds the `event` and `eventProperties` props to some native-ui components)